bin/check-iowait.rb

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env ruby
+#
+# check-iowait
+#
+# DESCRIPTION:
+#   Measures the IO wait. This is somehow different from what is achieved by iostat
+#
+# OUTPUT:
+#   plain text
+#
+# PLATFORMS:
+#   Linux
+#
+# DEPENDENCIES:
+#   gem: sensu-plugin
+#
+# USAGE:
+#   check-iowait.rb -c 30 -w 20 -s 6
+#
+# NOTES:
+#   N/A
+#
+# LICENSE:
+#   Hanine HAMZIOUI <hanynowsky@gmail.com> - Magic Online
+#   November 2016
+#   Released under the same terms as Sensu (the MIT license); see LICENSE
+#   for details.
+#
+
+require 'sensu-plugin/check/cli'
+
+#
+# Check IO WAIT
+#
+class CheckIOWAIT < Sensu::Plugin::Check::CLI
+  option :warning, long: '--warning WARNING', short: '-w WARNING', description: 'warning percentage', proc: proc(&:to_f), default: 50
```

Comment:
The alignment is very subtle it must be perfectly aligned with the first char (`:`) meaning that its gonna be some number of tabs and a final space.